modules/ossm-config-otel-kiali.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel-kiali_{context}"]
+= Configuring {DTProductName} with {KialiProduct}
+
+After {KialiProduct} is integrated with {DTProductName}, you can view distributed traces in the Kiali console. Viewing these traces can provide insight into the communication between services within the service mesh, helping you understand how requests are flowing through your system and where potential issues might be.
+
+.Prerequisites
+
+* You installed {SMProductName}.
+
+* You configured {DTShortName} with {SMProductName}.
+
+.Procedure
+
+. Configure the `Kiali` custom resource to access traces from the {TempoName} frontend:
++
+.Example `Kiali` custom resource configuration for tracing
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+spec:
+# ...
+external_services:
+  grafana:
+    enabled: true # <1>
+    in_cluster_url: "http://grafana-istio-system.apps-crc.testing/"
+    url: "http://grafana-istio-system.apps-crc.testing/"
+  tracing:
+    enabled: true
+    provider: tempo
+    use_grpc: false
+    in_cluster_url: http://tempo-sample-query-frontend.tempo:3200
```

Comment:
We might want to include the Tempo format URL when Tempo was configured in multi tenancy, something like: 

<3> In case the tempo resource was deployed in multiple tenant mode, the in_cluster_url config should point to the gateway with the tenant name as part of the url: http://tempo-resource-gateway.tempo:8080/api/traces/v1/{tenantName}/tempo